ballsdex/packages/admin/cog.py

```diff
@@ -1738,3 +1739,125 @@ async def user(
+                f"The amount of {settings.currency_name} " "to add cannot be negative.",
+                ephemeral=True,
+            )
+            return
+
+        if user_id is not None:
+            try:
+                user = await self.bot.fetch_user(int(user_id))
+            except ValueError:
+                await interaction.response.send_message(
+                    "The user ID you provided is not valid.", ephemeral=True
+                )
+                return
+            except discord.NotFound:
+                await interaction.response.send_message(
+                    "The given user ID could not be found.", ephemeral=True
+                )
+                return
+
+        assert user
+        player, created = await Player.get_or_create(discord_id=user.id)
+        await player.add_coins(amount)
+        await interaction.response.send_message(
+            f"Added {amount} {settings.currency_name} to {user.name}.", ephemeral=True
+        )
+
+        await log_action(
+            f"{interaction.user} added {amount} {settings.currency_name} "
+            f"to {user.name} ({user.id}).",
+            self.bot,
+        )
+
+    @coins.command()
+    @app_commands.checks.has_any_role(*settings.root_role_ids)
+    async def remove(
+        self,
+        interaction: discord.Interaction,
+        user: discord.User | None = None,
+        user_id: str | None = None,
+        amount: int = 0,
+    ):
+        """
+        Remove coins from a user.
+
+        Parameters
+        ----------
+        user: discord.User | None
+            The user you want to remove coins from.
+        user_id: str | None
+            The ID of the user you want to remove coins from.
+        amount: int
+            The number of coins to remove.
+        """
+        if (user is None and user_id is None) or (user is not None and user_id is not None):
+            await interaction.response.send_message(
+                "You must provide either `user` or `user_id`", ephemeral=True
+            )
+            return
+
+        if amount < 0:
+            await interaction.response.send_message(
+                f"The amount of {settings.currency_name} " "to remove cannot be negative.",
+                ephemeral=True,
+            )
+            return
+
+        if user_id is not None:
+            try:
+                user = await self.bot.fetch_user(int(user_id))
+            except ValueError:
+                await interaction.response.send_message(
+                    "The user ID you provided is not valid.", ephemeral=True
+                )
+                return
+            except discord.NotFound:
+                await interaction.response.send_message(
+                    "The given user ID could not be found.", ephemeral=True
+                )
+                return
+
+        assert user
+        player, created = await Player.get_or_create(discord_id=user.id)
+        await player.add_coins(amount)
```

Comment:
```suggestion
        await player.remove_coins(amount)
```
otherwise /remove adds coins